tests/unit/electionguard_gui/test_election_dto.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime, timezone
+
+from electionguard_gui.models.election_dto import ElectionDto
+from tests.base_test_case import BaseTestCase
+
+
+class TestElectionDto(BaseTestCase):
+    """Test the ElectionDto class"""
+
+    def test_get_status_with_no_guardians(self) -> None:
+        # ARRANGE
+        self.mocker.patch(
+            "electionguard_gui.models.election_dto.utc_to_str",
```

Comment:
This demonstrates how to mock a function that's imported in the method under test.  This string is non-intuitive and took me a while to figure out.  It's the method under test then the name of the function to mock.